src/client/clientService.ts

```diff
@@ -93,7 +96,7 @@ function createClientService({
     }),
     ...createIdentitiesRPC({
       identitiesManager,
-      gestaltGraph,
+      sigchain,
```

Comment:
Was surprised that `sigchain` wasn't part of the client service yet. That seems like a big oversight.

Reply:
I think it was part of the client service at some point but was later removed? I don't remember what it was previously used for though.

Reply:
This is an interesting point. I couldn't quite remember why `sigchain` wasn't already there (even for node claims which are already implemented). Looking back, it's because the sigchain is injected into the `NodeManager`, and it makes a call through `NodeManager` to perform the claiming process (i.e. `NodeManager.claimNode`). This call requires a `NodeConnection`, hence the current requirements on `NodeManager`.